data/meson.build

```diff
@@ -1,44 +1,34 @@
+icon_sizes = ['16', '24', '32', '48', '64', '128']
 
-# Install application icons.
```

Comment:
As mentioned previously (? It's strange how these comments show up...), I'd like for these comments to stay.

Reply:
> It's strange how these comments show up...

Strange in what?

Reply:
In the actual view of the diff between the files, this comment shows up on an earlier file, but on this timeline it shows it afterwards. You can more or less ignore the stuff in the parentheses 😆

Reply:
Oh okay